magpie/api/exception.py

```diff
@@ -113,8 +113,8 @@ def verify_param(   # noqa: E126
             raise TypeError("'paramCompare' cannot be 'None' with specified test flags")
         if isEqual or notEqual:
             # allow 'different' string literals for comparison, otherwise types must match exactly
-            if not (isinstance(param, six.string_types) and
-                    isinstance(paramCompare, six.string_types)) and type(param) != type(paramCompare):
+            if not ((isinstance(param, six.string_types) and isinstance(paramCompare, six.string_types))
+                    and type(param) != type(paramCompare)):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using type() instead of isinstance() for a typecheck.](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)